tfjs-backend-webgpu/src/from_pixels_webgpu.ts

```diff
@@ -33,8 +34,14 @@ export class FromPixelsProgram implements WebGPUProgram {
   constructor(outputShape: number[], useImport = false) {
     this.outputShape = outputShape;
     this.dispatchLayout = flatDispatchLayout(this.outputShape);
+    util.assert(
+        this.outputShape.length >= 1,
+        () => `outputShape.length of FromPixelsProgram is ${
+            this.outputShape.length}`);
+    this.workPerThread = this.outputShape[this.outputShape.length - 1];
```

Comment:
Is it better to directly pass `numChannels` to this constructor for understanding? So you can directly use this.workPerThread = numChannels or `this.dispatch = computeDispatch(
        this.dispatchLayout, this.outputShape, this.workGroupSize,
        [numChannels , 1, 1]);`

Reply:
done